infra/gcp/ensure-conformance-storage.sh

```diff
@@ -90,5 +94,51 @@ for REPO; do
     # Enable writers on the bucket
     color 6 "Empowering ${BUCKET_WRITERS} to GCS"
     empower_group_to_write_gcs_bucket "${BUCKET_WRITERS}" "${BUCKET}"
+
+    readonly SERVICE_ACCOUNT_NAME="service-${REPO}"
```

Comment:
nit: I would either
- drop the `service-` prefix since it's too generic
- rename to `${REPO}-writer` to hint at what this account is used for

(I don't care strongly enough to block if you ignore this though)

Reply:
So I used this pattern because I deduced from #604 these pattern was used for `capi-openstack`, `cri-o`, `huaweicloud` already and changing it would require some work, but I'm open to changing it (I also prefer the way you suggested it). @dims wdyt?

Reply:
if we're stuck with the names so be it; per @dims' question do you need the existing keys?

Reply:
I think it would be good to have it consistent if for example we would want to change something in the future

Reply:
note from slack `And also if you could look once again here: https://github.com/kubernetes/k8s.io/pull/873 as we discussed this yesterday and we can proceed without storing these secrets (which were currently given to three projects)`